src/index.js

```diff
@@ -0,0 +1,338 @@
+import {
+  setListItemsStorage,
+  setListStorage,
+  getLocalStorage,
+  setLocalStorage,
+  listItemsStorage,
+  updateLocalStorage,
+  listStorage,
+  listLength,
+} from './storage';
+
+
+const projectInput = document.getElementById('project');
+const createProject = document.getElementById('create-project');
+const ul = document.querySelector('.list-group');
+const rightBtn = document.getElementById('slide-right');
+const leftBtn = document.getElementById('slide-left');
```

Comment:
- [ ] [ ] Let's try to put all these DOM selections in the file and DOM related functions inside a Dom Module in another file.
Read more about the module pattern [on the Odin project.](https://www.theodinproject.com/courses/javascript/lessons/factory-functions-and-the-module-pattern)